### PR TITLE
RP-31: First draft of a scheduler.

### PR DESCRIPTION
**Pull request by DeeVeX**
That's how the scheduler will work.
It's only possible to schedule 1 job. The job will trigger a hard coded function defined in the scheduler.py file.

Easy to modify this draft to manage multiple jobs and use service/action/topic calls instead of functions, so calls can be dynamic and defined in the configdb. (If required)

### CODE REVIEW

File: src/robopatrol/scheduler.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+import time
+import requests
+import json
+from apscheduler.schedulers.background import BackgroundScheduler
+
+poll_interval_sec = 5
+my_job_interval_sec = 3600
+my_job_name = 'my_job'
+configURL = 'http://192.168.115.1:9998/schedule'
```
Comment (chilloutman) [inline src/robopatrol/scheduler.py]: This should be localhost by default.

File: src/robopatrol/scheduler.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+def my_job():
+    print('Tick! The time is: %s' % datetime.now())
```
Comment (chilloutman) [inline src/robopatrol/scheduler.py]: Here we should get the current job config and check that the job config was not deleted.

We should also de-schedule the job in the while-loop below.

**Comment by chilloutman** [comment]
Waiting for this PR to be reviewed: https://github.com/robopatrol/robopatrol-server/pull/2

**Comment by chilloutman** [comment]
Ci is broken for this module. We should fix that... Can anyone other than @sidloki do that? :->

**Comment by DeeVeX** [comment]
This PR is ready to get tested. If you need any assistance - just ask ;-)

**Comment by chilloutman** [comment]
@DeeVeX Please rebase/merge with master to fix the CI. And then please assign someone for review/testing.

**Comment by DeeVeX** [comment]
I created an entry in our wiki for the scheduler:
https://github.com/robopatrol/robopatrol/wiki/Scheduler

If there are any problems, please do not hesitate to contact.

**Comment by joshfreak7** [comment]
Successfully tested.

**Pull request merged**